Packs/Core/Playbooks/playbook-XCloud_Cryptomining.yml

```diff
@@ -0,0 +1,786 @@
+          right:
+            value:
+              simple: userVerification
+          ignorecase: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 720,
+          "y": 830
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "54":
+    id: "54"
+    taskid: 741176fb-7a63-4d03-85aa-6cc387b18714
+    type: regular
+    task:
+      id: 741176fb-7a63-4d03-85aa-6cc387b18714
+      version: -1
+      name: Close incident as True Positive
+      description: commands.local.cmd.close.inv
+      script: Builtin|||closeInvestigation
+      type: regular
+      iscommand: true
+      brand: Builtin
+    nexttasks:
+      '#none#':
+      - "43"
+    scriptarguments:
+      closeReason:
+        simple: Resolved - True Positive
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 1310,
+          "y": 2090
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "55":
+    id: "55"
+    taskid: 35512734-45a5-4d02-8ee1-b9f0fc678ea4
+    type: playbook
+    task:
+      id: 35512734-45a5-4d02-8ee1-b9f0fc678ea4
+      version: -1
+      name: Handle False Positive Alerts
+      playbookName: Handle False Positive Alerts
+      type: playbook
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "43"
+    scriptarguments:
+      ShouldCloseAutomatically:
+        complex:
+          root: inputs.ShouldCloseAutomatically
+      ShouldHandleFPautomatically:
+        complex:
+          root: inputs.ShouldHandleFPautomatically
+      alertName:
+        complex:
+          root: alert
+          accessor: name
+      sourceIP:
+        complex:
+          root: alert
+          accessor: hostip
+      username:
+        complex:
+          root: alert
+          accessor: username
+    separatecontext: true
+    continueonerrortype: ""
+    loop:
+      iscommand: false
+      exitCondition: ""
+      wait: 1
+      max: 100
+    view: |-
+      {
+        "position": {
+          "x": 720,
+          "y": 1370
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+view: |-
+  {
+    "linkLabelsPosition": {
+      "40_41_yes": 0.52,
+      "40_54_#default#": 0.57,
+      "41_43_#default#": 0.42,
+      "41_54_Yes": 0.42,
+      "49_27_Malicious": 0.62,
+      "49_29_User Verification": 0.61
+    },
+    "paper": {
+      "dimensions": {
+        "height": 2535,
+        "width": 970,
+        "x": 720,
+        "y": -200
+      }
+    }
+  }
+inputs:
+- key: incident_id
```

Comment:
Why do you need it? and if we do, lets add a default field (as far as I know, the incident ID of the alert doesnt exist in the alert context.

Reply:
Correct, Removed.